lib/config/migration.ts

```diff
@@ -53,6 +53,12 @@ export function migrateConfig(
               (item) => item !== 'prEditNotification'
             );
         }
+      } else if (key === 'matchStrings' && is.array(val)) {
+        migratedConfig.matchStrings = val.map((matchString) => {
+          return is.string(matchString)
+            ? matchString.replace(regEx(/<?lookupName>/g), '<?packageName>')
```

Comment:
```suggestion
            ? matchString.replace(regEx(/\\?<lookupName>/g), '?<packageName>')
```

Shouldn't the `?` before? see snapshots above, they look wrong